### PR TITLE
MRG+1: fix tfr docs

### PR DESCRIPTION
**Pull request by jona-sassenhagen**
A few obvious fixes to the docs - mostly stuff left over after introduction of EpochsTFR I think.

@kingjr

### CODE REVIEW

File: mne/time_frequency/tfr.py [hunk 1]
```diff
@@ -654,8 +654,8 @@ def tfr_morlet(inst, freqs, n_cycles, use_fft=False, return_itc=True, decim=1,
     n_jobs : int, defaults to 1
         The number of jobs to run in parallel.
     picks : array-like of int | None, defaults to None
-        The indices of the channels to plot. If None, all available
-        channels are displayed.
+        The indices of the channels to work on. If None, all available
```
Comment (kingjr) [inline mne/time_frequency/tfr.py]: 'on which to work'; or better 'to decompose'

**Comment by kingjr** [comment]
LGTM apart from this

File: mne/time_frequency/tfr.py (hunk 1 above)
Comment (jona-sassenhagen) [inline mne/time_frequency/tfr.py]: Yeah, decompose.

**Comment by Eric89GXL** [comment]
Also killed irrelevant CIs here

**Comment by Eric89GXL** [comment]
LGTM, feel free to merge once the Travis 2.7-nodata (style checking) build comes back happy

**Comment by Eric89GXL** [comment]
Thanks @jona-sassenhagen

**Pull request merged**